File: library/src/main/java/com/dimeno/permission/callback/AbsPermissionCallback.java
package com.dimeno.permission.callback;

/**
 * abstract implementation of {@link PermissionCallback}
 * Created by wangzhen on 2020/4/16.
 */
public abstract class AbsPermissionCallback implements PermissionCallback {
    @Override
    public void onNotDeclared(String[] permissions) {

    }
}

File: library/src/main/java/com/wangzhen/permission/callback/AbsPermissionCallback.kt
package com.wangzhen.permission.callback

/**
 * abstract implementation of [PermissionCallback]
 * Created by wangzhen on 2020/4/16.
 */
abstract class AbsPermissionCallback : PermissionCallback {
    override fun onNotDeclared(permissions: Array<String>) {}
}